instrumentation/spring/spring-boot-autoconfigure/testing/build.gradle.kts

```diff
@@ -18,3 +18,13 @@ dependencies {
   compileOnly("io.opentelemetry:opentelemetry-sdk-testing")
   compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi")
 }
+
+tasks {
```

Comment:
I'd guess that this isn't needed after https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/15654 that adds `-Xlint:-classfile`